feature_engine/selection/recursive_feature_elimination.py

```diff
@@ -88,6 +88,9 @@ class RecursiveFeatureElimination(BaseEstimator, TransformerMixin):
         A dictionary containing the feature, and the change in performance incurred
         when training a model without that feature.
 
+    selected_features_: list
+        The selected features by the recursive feature elimination selector
+
 
```

Comment:
I'll include this modification in #184 could we remove this file from here? I modified it a lot in mentioned PR, so we are going to have some conflicts otherwise.  Thanks for pointing this out!

Thank you!